src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -157,65 +157,66 @@ private static double CalculatePercentDiff(Bitmap image1, List<Tuple<int,int,Col
 
 #endregion
 
-#region meshes
-        [Test]
-        public void StandardMeshGeometryRender()
-        {
-            OpenVisualizationTest(@"imageComparison\spherecolors.dyn");
-            RunCurrentModel();
-            RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
-        }
+        #region meshes
```

Comment:
@QilongTang can you remind me? Do I have my white space settings incorrect? Or does @zeusongit?

Reply:
> @zeusongit can you try building your branch here:
> https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/WpfVisualizationTests_OnPerfMachine/build?delay=0sec
> 
> this will run your branch and the wpf visualization tests (including your new image comparison one) on the performance CI machine with a nvidia GPU.

It seems I dont have the Job/Build permissions

Reply:
This is interesting.. Two issues here:
1. I think @zeusongit here is using same VS default setting which `#region` will start with  indents but unlike `#if` which will start first of lines. My local defaults is same with @zeusongit.
2. However all the function body below should not require additional indents.. You can check lots of existing example code in Dynamo. @zeusongit You amy want to simply select all lines in this file and do a format all to correct them..